_posts/randy/2022-06-24-reflecting-on-the-future.md

```diff
@@ -0,0 +1,68 @@
+
+- **Precision** - measure of the stability of the instrument and its capability of resulting in the same measurement over and over again for the same input signal 
+
+- **Noise** - uncertainty of the measurement because of the effect of noise in the measurement and is factored into determining the accuracy 
+
+The different spellings of the key word Analog (US) and Analogue (UK) could loosely be used as an example of analog vs. digital representation. The original word spelling Analogue is all/whole: analog; upon digital conversion, the unneeded data (u & e) is silent and not encoded. On playback, the audible result is the same – with Digital Analog coming in with a 25% letter saving. 
+
+Starting from ground zero; my technology perspective comes from growing up in the 1960s, especially memories of visiting my grandparents on the farm. My mother's father was an early century homesteader – by the Roaring Twenties he had not a little house prairie and all the best of everything. 
+
+Then 1929 crashed a lot of dreams; it seemed the family house and occupants got stuck in that time.  
+
+The same big wooden box telephone and party line was still in use into the 60s. Their communication technology did not change for over 40 years. No one needed anything more than what they knew about at the time. 
+
+We now live in a world where technology does not stay relevant for 40 years. While CDs are still going since 1983 and DVDs since 1997, it's only because we can't save ourselves from dipping into the Dollar Store Disc Bin. Being in any Digital business means that anyone can replicate your data, your product, your idea - at less cost to steal than it took you to make it. If you don't make it big fast, you are the one in the Dollar bin. In many industries, constant product churn is necessary just to keep even of the game. 
+
+My present perspective is that I need to spend a lot more of my present tense time envisioning what my influence on the world's direction could be, so I can work more effectively now to radiate positive energy in the right direction. 
+
+Amid plenty of success, significant portions of my used-up lifetime and hard-earned money were wasted chasing down a few wrong avenues and dreams. Bleeding edge learning came with a costly curve - the urge has long hoped to bury those bodies and cover the trail leading to them. My family has instead encouraged me to go back inside my head and make sense of those experiences - good and bad, as they relate to our family business today. 
+
+Analog Sampling (or should it be Analog Dumpling: a bite sized pinch-wrapped morsel?) will thus forward be a digital (online) snippet of some analog (offline) part of the world as it happened to me. You probably don't need to hear the TLDR versions, so for the time sensitive audience we'll set the Frequency at weekly and the Amplitude at the 1,024-word stun level. 
```

Comment:
Ah, I had read this paragraph I think before you made some edits.  This last bit makes a lot more sense now.